pygitops/operations.py

```diff
@@ -9,6 +9,7 @@
 
 from pygitops._util import checkout_pull_branch as _checkout_pull_branch
 from pygitops._util import get_lockfile_path as _get_lockfile_path
+from pygitops._util import is_git_repo as _is_git_repo
```

Comment:
I like that you thought about this! id agree that this is not a type of operation that we want to expose to our users